blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -24,15 +24,15 @@ import { calculateNode } from '../util/KaraokeHelper';
 
 const { string, object, any, func } = PropTypes;
 
-const queuedState = () => (
+const QueuedState = () => (
     <EmptyStateView tightSpacing>
         <p>
             <Icon {...{
                 size: 20,
                 icon: 'timer',
                 style: { fill: '#fff' },
             }} />
-            <span>Waiting for run to start.</span>
```

Comment:
I'm not sure if this ever worked, I have never seen this copy show up with pipeline... is this the right thing to change?